ykrt/src/compile/jitc_yk/jit_ir.rs

```diff
@@ -413,6 +417,8 @@ instr!(StoreGlobal, StoreGlobalInstruction);
 instr!(LoadArg, LoadArgInstruction);
 instr!(Call, CallInstruction);
 instr!(PtrAdd, PtrAddInstruction);
+// FIXME: Use a macro for all binary operations?
```

Comment:
I was gonna say, we are going to want a macro for these when we start adding more.

Reply:
(not now though)

Reply:
Either that or we make a `BinopInstruction(op1, op2, binop)`.

Reply:
Right, this was why I was advocating for a single binary operation instruction: you'd have one handler instead of 17 (the handler would contain a match over a `BinOpKind`).

Let's leave it as-is for now though.

Reply:
Let's not relitigate one vs. many bin ops. It's a very minor issue either way.